packages/foreman/foreman/foreman.spec

```diff
@@ -830,6 +809,15 @@ Meta package with support for plugins.
 %{_sysconfdir}/rpm/macros.%{name}-plugin
 %{_datadir}/%{name}/schema_plugin.rb
 
+%package build
+Summary: Foreman package RPM support
+Group: Development/Libraries
+
+%description build
+Meta package with support for building RPMs in the Foreman release cycle.
+
+%files build
+%{_sysconfdir}/rpm/macros.foremandist
```

Comment:
It used to be `%{_sysconfdir}/rpm/macros.%{name}-dist` in `-release` Should we keep it like that?